crates/ruff/resources/test/fixtures/pyflakes/F541.py

```diff
@@ -33,3 +33,5 @@
 # Errors
 f"{v:{f'0.2f'}}"
 f"{f''}"
+f"{{test}}"
+f'{{ 40 }}'
```

Comment:
What about cases like...

```py
>>> f"\{{x}}"
'\\{x}'
>>> f"{{a {{x}}"
'{a {x}'
>>> f"{{{{x}}}}"
'{{x}}'
```

Any others to consider? (These may all be handled appropriately.)

Reply:
Interesting. The second and third case are handled correctly. The first gives a parsing error:
`Error: f-string: single '}' is not allowed at line 1 column 8`